### ISSUE TITLE
UART receive MAGIC_STOP_BYTE before end of message

### ISSUE THREAD

**Issue by eirikstor** [vortexntnu/mcu-firmware#10]
Receiving the byte 0x40 (MAGIC_STOP_BYTE) in the data payload will stop the message from being received. For example: sending the number 1600 as two bytes is done by sending 0b00000110 (0x6), and then 0b01000000 (0x40). The receiving MCU will therefore think that the message is over and disregard the rest of the message.

**Comment by sindrehan** [comment]
Why is there a need for a stop byte? Isn't the  message type specified in the start of the message? With the start byte and the message type we should know how many bytes to expect and when the message ends.

Or am I missing something?

**Comment by eirikstor** [comment]
Indeed! No you are not missing something. Will fix, thank you :smile:
